app/services/builders/blog_metric_chart/growth_month_over_month.rb

```diff
@@ -0,0 +1,27 @@
+module Builders
+  module BlogMetricChart
+    class GrowthMonthOverMonth < Builders::BlogMetricChart::Base
+      def initialize(months = 13)
+        super(months + 1)
+      end
+
+      private
+
+      def process_data(metrics_data)
+        metrics_data.each.inject({}) do |hash, key_value_pair|
+          month, metric_value = key_value_pair
```

Comment:
Another way to accomplish the same could be... 

```suggestion
        metrics_data.each_with_object({}) do |hash, (month, metric_value)|
```

WDYT?

Reply:
I thought I couldn't do that. Thank you!